quad_name: 'hummingbird'
log_dir: '/tmp/test'
ideal_inputs: False
seq_len: 1
verbose: False
throw_params:
  vlin_range: 5
  vang_range: 5
  pos_range: 5
checkpoint:
  resume_training: True
  resume_file: "results/train_fma_cork_seq_1_fts_tracks_newcam_new_end/20240620-150939/train/ckpt-29"
test_time:
  max_rollouts: 5
  min_number_fts: 40
  execute_nw_predictions: True
  fallback_threshold_thrust: 100
  fallback_threshold_rates: 100
  use_imu: True
  use_fts_tracks: True
